app/templates/pdf/event_invoice.html

```diff
@@ -167,8 +167,8 @@ <h1>INVOICE {{ invoice.identifier }}</h1>
         {% if user.billing_tax_info %}
           <div><span>TAX ID</span> {{user.billing_tax_info or ''}}</div>
         {% endif %}
-        <div><span>DATE</span> {{ invoice.issued_at | date }}</div>
-        <div><span>DUE DATE</span> {{ invoice.due_at | date }}</div>
+        <div><span>DATE</span> {{ invoice.issued_at | date(event.timezone) }}</div>
+        <div><span>DUE DATE</span> {{ invoice.due_at | date(event.timezone) }}</div>
```

Comment:
The date filter is not timezone aware. It'll fail. Please test the changes before asking for review. Reduce the request review cycle please. The changes on this PR either go opposite of what it should be or plain don't work.

Why don't you test it first?

Reply:
I didn't requested for review this tym. I will test and then ask for review and will make sure to reduce the request review cycle. Sorry for the inconvinience